frontend/app/src/people/widgetViews/OrganizationView.tsx

```diff
@@ -298,71 +267,11 @@ const Organizations = (props: { person: Person }) => {
                 borderRadius: '50%'
               }}
             >
-              <Formik
```

Comment:
is formkit no longer a dependency in this project if this is removed?

Reply:
Nope, I doubt that.

It was used to render the forms on this page previously, I removed it because I no longer need it